scripts/diagnostics-agent-setup.ps1

```diff
@@ -51,7 +52,13 @@ function Get-DCOSVersionFromFile {
 }
 
 function Get-MonitoredServices {
-    $services = @('dcos-diagnostics', 'dcos-mesos-slave', 'dcos-metrics')
+    Param(
+        [bool]$AddMetricsService
+    )
+    [System.Collections.ArrayList]$services = @('dcos-diagnostics', 'dcos-mesos-slave')
+    if ($AddMetricsService) {
```

Comment:
Since `$IncludeMetrics` is already a global parameter to the script, why not use it directly here ?

Instead of using extra `$NeedToMonitorMetricsService` and `$AddMetricsService` functions parameters, which are simply reusing the value of a global parameter accessible everywhere in the script.